src/components/Projects/ProjectCard/ProgressBarTooltip.tsx

```diff
@@ -8,31 +8,33 @@ import Time, { formatDate } from '../../../utils/date/Time'
 import { CLIFF_PERIOD_IN_DAYS } from '../../../utils/proposal'
 
 interface Props {
-  grant: ProposalProject
+  proposalProject: ProposalProject
   isInCliff: boolean
   children: React.ReactNode
 }
 
-function ProgressBarTooltip({ grant, isInCliff, children }: Props) {
+function ProgressBarTooltip({ proposalProject, isInCliff, children }: Props) {
   const t = useFormatMessage()
   const intl = useIntl()
 
-  const { contract, tx_amount, token, enacted_at, enacting_tx } = grant
-  const isOneTimePayment = !contract && enacting_tx
-  const vestedAmount = (contract ? contract.vested_amount : tx_amount) || 0
-  const releasedAmount = !isOneTimePayment && contract ? contract.released : 0
+  const { funding } = proposalProject
+  const { vesting, one_time_payment, enacted_at } = funding!
```

Comment:
Isn't it possible funding is not available here? I'd prefer not to `!` and safe check everything.